cypress/e2e/Tests/Admin/GX3-3039-findUser.cy.js

```diff
@@ -0,0 +1,83 @@
+import { removeLogs } from '@helper/RemoveLogs';
+import { userManagementPage } from '@pages/Admin/userManagement.Page';
+import data from '../../../fixtures/data/gx3-3039-findUser.json';
+
+const { username, password } = Cypress.env('AdminUser');
+
+describe('GX3-3039 | OrangeHRM | Admin | Buscar un usuario', () => {
+	beforeEach('PRC: Login successfully, navegate to Admin section, and there is a user to search', () => {
+		cy.visit('/');
+		cy.url().should('contain', 'login');
+
+		cy.Login(username, password);
+		cy.visit('/admin/viewSystemUsers');
+		cy.url().should('contain', 'viewSystemUsers');
+		//! La siguiente assertion falla cuando el navegador cambia a idioma español... a veces ocurre
+		//! cy.contains(data.userManagement).should('be.visible');
+	});
+
+	it('GX3-3068 | TC01: Should successfully filter users by an existing "Username"', () => {
+		userManagementPage.searchUserSuccessfully(username);
+		cy.contains(data.recordFound).should('be.visible');
+		userManagementPage.get.searchResultTable().contains(username);
+	});
+
+	it.skip('GX3-3068 | TC02: Should not filter users with a non-existent "Username"', () => {
+		userManagementPage.searchUserSuccessfully(data.nonExistentUsername);
+		cy.contains(data.noRecordsFound).should('be.visible');
+		userManagementPage.get.searchResultTable().should('be.empty');
+	});
+
+	//! Filtra mal: cuando se selecciona el rol Admin, a veces se muetran usuarios con rol ESS
```

Comment:
Esto es muy peculiar, yo creo que depende del negocio si es bug o no. Aca lo que sucede es que cuando seleccionar alguno de las opciones de los dropdowns tenes que hacer click en search para que haga la busqueda cosa que no sucede cuando ingresas texto en los inputs.
Por mi parte si califica como bug pero es relativo

Reply:
recorda que en caso de ser bug, este test se debe skippear de la siguiente forma `it.skip` seguido de notificar al equipo.Ya dependiendo si sos hibrido o qa varia quien debe reportar el bug y por ultimo queda decidir si dejamos esta TD para cuando se resuelva el bug (bloqueante) o hacer otra TD para habilitarlo en caso de no ser critico y poder cerrar la tarea